pkg/minikube/assets/addons.go

```diff
@@ -724,9 +724,17 @@ func GenerateTemplateData(addon *Addon, cfg config.KubernetesConfig) interface{}
 				"image": strings.Split(image, "@")[0],
 			})
 		} else if opts.ImageRepository != "" {
+			tmp := strings.Split(image, "/")
+			var realImage string
```

Comment:
for code readability do u mind commenting on this what it means RealImage ? 
preferably with a link to a github issue that why we do this ?

Reply:
As I described above.

> When I use the --image-mirror-country=cn flag to start a cluster, the storage-provisioner pod will use the
registry.cn-hangzhou.aliyuncs.com/google_containers/k8s-minikube/storage-provisioner:v4 image, but the correct image should be
registry.cn-hangzhou.aliyuncs.com/google_containers/storage-provisioner:v4

the real image should `storage-provisioner:v4`, not `k8s-minikube/storage-provisioner:v4` when using `--image-mirror-country=cn` flag. @medyagh

Reply:
@scaat would u plz add the comment in the code ?
how about a unit test for this ?